templates/landing/landing-dashboard.html

```diff
@@ -1,9 +0,0 @@
-{{ define "content" }}
```

Comment:
@dpham273 this was a wrong file to use.

I already had a html file at `/templates/userdash/userdash-top.html` and was intended to use that.

Notice inside the `templates` folder, each subfolder serves a different purpose. The `landing` templates are only for locations that doesn't require authentication. Please keep it that way. And then the `userdash` folder is supposed to be used for userdash.